modules/rtpengine/rtpengine.c

```diff
@@ -1957,12 +2027,23 @@ static bencode_item_t *rtpp_function_call(bencode_buffer_t *bencbuf, struct sip_
 			}
 			LM_ERR("proxy replied with error: %.*s\n", error.len, error.s);
 		}
-        goto error;
+		goto error;
 	}
 
 	if (body_out)
 		*body_out = body;
 
+	if (op == OP_DELETE) {
+		/* Delete the key<->value from the hashtable */
+		if (!rtpengine_hash_table_remove(&callid)) {
```

Comment:
Some notes:

  * when session is created, there is no to-tag (initial invite)
  * the via-branch can be different for re-invite or bye, so it cannot be used as part of the key all the time